rpc/rpc-lib/src/protocol-common.h

```diff
@@ -315,6 +315,12 @@ enum glusterd_mgmt_v3_procnum {
     GLUSTERD_MGMT_V3_MAXVALUE,
 };
 
+enum gf_fd_status {
+    FD_REOPEN_NOT_ALLOWED = 0,
+    FD_REOPEN_ALLOWED,
```

Comment:
It is better to have the default value be zero, so REOPEN_ALLOWED as 0

Reply:
This might not be good for code reading. The key is 'is-reopen-allowed-on-fd' which sounds to be expecting a boolean value. If we say 0 means reopen allowed, that might lead to confusion.

Reply:
fd status doesn't seem to be boolean if it has 3 values. Maybe the key is wrong in that case?

Reply:
Does 'local->fd_status' instead of local->is_reopen_allowed and key 'fd-status-for-reopen' sounds good?

Reply:
local->fd_reopen_status would probably be better. Even the enum type also

Reply:
enum type gf_fd_reopen_status

Reply:
Changed the enum, key in dict & local variable. Made REOPEN_ALLOWED as  default.